<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loading_editorMainWindow</class>
 <widget class="QMainWindow" name="Loading_editorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="Loading_editor_tab_images_import">
      <property name="styleSheet">
       <string notr="true">        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="svg_icon_frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>320</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply_icon_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">     QLabel {
        font-family: Sergo UI;
        color: #9D9D9D;
    }

    /* QPushButton default and hover styles */
    QPushButton {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }

    /* QPushButton default and hover styles */
    QPushButton:disabled {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #8c8c8c;
        background-color: #1C1C1C;
    }</string>
         </property>
         <property name="text">
          <string>Apply Icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="PlainTextEdit_Description_2">
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">    QPlainTextEdit {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding: 4px;
        padding-left: 6px;
        padding-right: 6px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }



    QPlainTextEdit:focus {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding: 4px;
        padding-left: 6px;
        padding-right: 6px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPlainTextEdit:hover {
    }
    QPlainTextEdit:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;
        font: 580 9pt &quot;Segoe UI&quot;;

    }</string>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>A community map created by:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="apply_images_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:10pt; font-weight:580; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Apply images&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Applies all images above to the loading screen of the selected addon. &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Be careful: old images stored in &lt;span style=&quot; font-family:'Courier New';&quot;&gt;game\addons\%addon_name%\panorama\images\map_icons\screenshots\1080p\&lt;/span&gt; will be deleted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLabel {
        font-family: Sergo UI;
        color: #9D9D9D;
    }

    /* QPushButton default and hover styles */
    QPushButton {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
      </property>
      <property name="text">
       <string>Apply images</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="apply_description_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">    QLabel {
        font-family: Sergo UI;
        color: #9D9D9D;
    }

    /* QPushButton default and hover styles */
    QPushButton {

        font: 580 10pt &quot;Segoe UI&quot;;
        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:18px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
      </property>
      <property name="text">
       <string>Apply description</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
